assets/scss/validation.scss

```diff
@@ -1,4 +1,4 @@
-.form-group.validation-error {
+.govuk-form-group.validation-error {
```

Comment:
Shouldn’t it be govuk-error-message?

Reply:
I have now removed this file as govuk-error styling is being pulled correctly from hof